Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/azavea/spark:1.6.1
+
+ENV VERSION 0.1.0
+
+COPY geop/target/scala-2.10/usace-programanalysis-geop-assembly-${VERSION}.jar /opt/geoprocessing/
+
+WORKDIR /opt/geoprocessing
+
+EXPOSE 8090
+
+ENTRYPOINT ["/bin/bash", "-c", "spark-submit /opt/geoprocessing/usace-programanalysis-geop-assembly-${VERSION}.jar"]
```

Comment:
We may want to change this to an `ENTRYPOINT` script so that we can more easily override options passed to `spark-submit` at container launch time:

```diff
$ git diff Dockerfile         
diff --git a/Dockerfile b/Dockerfile
index 68d8548..26805c3 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -2,10 +2,11 @@ FROM quay.io/azavea/spark:1.6.1
 
 ENV VERSION 0.1.0
 
-COPY geop/target/scala-2.10/usace-programanalysis-geop-assembly-${VERSION}.jar /opt/geoprocessing/
+COPY geop/target/scala-2.10/usace-programanalysis-geop-assembly-${VERSION}.jar /opt/geoprocessing/usace-programanalysis-geop.jar
+COPY scripts/docker-entrypoint.sh /opt/geoprocessing/
 
 WORKDIR /opt/geoprocessing
 
 EXPOSE 8090
 
-ENTRYPOINT ["/bin/bash", "-c", "spark-submit /opt/geoprocessing/usace-programanalysis-geop-assembly-${VERSION}.jar"]
+ENTRYPOINT ["./docker-entrypoint.sh"]
$ cat scripts/docker-entrypoint.sh 
#!/usr/bin/env bash

set -e

exec /opt/spark/bin/spark-submit "$@" usace-programanalysis-geop.jar 2>&1
```

This way we can launch the container image with a larger amount of driver memory, for example:

```bash
$ docker run --rm -p 4040:4040 quay.io/usace/program-analysis-geoprocessing:f98d17f --driver-memory 2G
```